app/views/admin/logs/new.slim

```diff
@@ -0,0 +1,9 @@
+div.modal-header
+  button.close type="button" data-dismiss="modal" aria-label== t(:close)
+    span aria-hidden="true" &times;
+  h4.modal-title = t("log.new_log")
+div.modal-body
+  = render "admin/logs/form"
+div.modal-footer
+  a.btn.btn-default data-dismiss="modal" = t(:cancel)
+  a.btn.btn-primary data-action="submit" = t("log.save")
```

Comment:
A lot of repetition here with the edit one. Only the title differs and that could easily be parameterized. I will combine these.